python/mlc_chat/chat_module.py

```diff
@@ -18,25 +18,6 @@
 # pylint: enable=line-too-long
 
 
-def quantization_keys():
-    r"""Get the keys of all quantization schemes."""
-    return [
-        "autogptq_llama_q4f16_0",
-        "q0f16",
-        "q0f32",
-        "q3f16_0",
-        "q3f16_1",
-        "q4f16_0",
-        "q4f16_1",
-        "q4f16_2",
-        "q4f16_ft",
-        "q4f32_0",
-        "q4f32_1",
-        "q8f16_0",
-        "q8f16_ft",
-    ]
```

Comment:
I don't see this being used anywhere. Is it safe to remove?

Reply:
Yep I think so.

Reply:
👍